secure/key-manager/src/lib.rs

```diff
@@ -147,13 +147,13 @@ where
                 Err(Error::LivenessError(last_value, current_value)) => {
                     // Log the liveness error and continue to execute.
                     let error = Error::LivenessError(last_value, current_value);
-                    info!(LogSchema::new(LogEntry::CheckKeyStatus)
+                    error!(LogSchema::new(LogEntry::CheckKeyStatus)
                         .event(LogEvent::Error)
                         .liveness_error(&error));
```

Comment:
Could this be temporary / go away?  Mostly, just trying to determine `error!` vs `warn!`

Reply:
Given the frequency at which the key manager checks the state of the keys and blockchain (which is minutes, at most), if this case is hit, it more than likely means consensus is down and something is really wrong. Or another way to look at it: I'd be really surprised if the timestamp on-chain hasn't increased at all in a space of minutes, so I think an error should be logged.